src/shim/umq/hwctx.cpp

```diff
@@ -11,7 +11,7 @@ namespace shim_xdna {
 
 hw_ctx_umq::
 hw_ctx_umq(const device& device, const xrt::xclbin& xclbin, const xrt::hw_context::qos_type& qos)
-  : hw_ctx(device, qos, std::make_unique<hw_q_umq>(device, 8), xclbin)
+  : hw_ctx(device, qos, std::make_unique<hw_q_umq>(device, 16), xclbin)
```

Comment:
I understand this 8 -> 16 is a temporary change due to the nop can flood the queue with simnow on slow machine, but 16 might be good enough or not.

We need to think a way to handle this rather than simply double the size. Let's discuss this before take actions.